src/main/java/org/ethereum/beacon/discovery/DiscoveryManagerImpl.java

```diff
@@ -93,6 +96,7 @@ public DiscoveryManagerImpl(
             outgoingPipeline,
             expirationSchedulerFactory);
     incomingPipeline
+        .addHandler(rateLimiter)
```

Comment:
I would consider moving the rate limiter even more close to the 'origin', e.g. here to the very head of Netty pipeline  https://github.com/ConsenSys/discovery/blob/2eedf4e98b6ffe12f455d74dca2a26eb50bf2c5b/src/main/java/org/ethereum/beacon/discovery/network/NettyDiscoveryServerImpl.java#L60

This way no our code would be affected by a possible naive DoS attack 

In the latter I would also check for existing Netty rate limit handlers (I thought I saw something like this)